spec/helpers/application_helper_spec.rb

```diff
@@ -41,4 +41,78 @@
       expect(helper.library_name('NOSUCHLIBRARY')).to eq 'NOSUCHLIBRARY'
     end
   end
+
+  describe '#render_checkout_status' do
```

Comment:
Looks like these tests are duplicated here and in `checkouts_helper_spec`.  Can we remove the one here in `application_helper_spec`?